pennylane/devices/qubit/sampling.py

```diff
@@ -330,7 +330,7 @@ def _process_single_shot(samples):
             prng_key=prng_key,
         )
     except ValueError as e:
-        if str(e) != "probabilities contain NaN":
+        if "robabilities contain NaN" not in str(e):  # capitalization might differ
```

Comment:
Possible alternative?
```suggestion
        if "probabilities contain nan" not in str(e).lower():
```